AdobeStockImage/Model/Storage.php

```diff
@@ -101,6 +103,19 @@ public function save(string $imageUrl, string $destinationDirectoryPath = '') :
         return $destinationPath;
     }
 
+    /**
+     * Generate image name by Title + id.
+     *
+     * @param aray $imageData
+     * @return string
+     */
+    private function generateImageName(array $imageData) :string
+    {
+        $imgName = str_replace(' ', '', $imageData['title']).$imageData['id'];
```

Comment:
let's replace spaces in the original name with dashes ('-') and shorten the file name to i.e. 32 characters (as titles can be really long)

Reply:
ok